src/controllers/logging/ChannelLog.cpp

```diff
@@ -9,7 +9,8 @@ ChannelLog::ChannelLog(QString channelName)
 
 QString ChannelLog::channelName() const
 {
-    return this->channelName_;
+    QString lowercaseChannelName_ = this->channelName_.toLower();
+    return lowercaseChannelName_;
```

Comment:
nit: You don't have to make a temporary variable here, you can just `return this->channelName_.toLower();` since [QString::toLower](https://doc.qt.io/qt-5/qstring.html#toLower) returns a copy of the string